sidebar.js

```diff
@@ -1,17 +1,15 @@
 let sidebarTemplateContent
 
-async function loadSidebarTemplate () {
-  const response = await fetch('./sidebar.html')
-  const text = await response.text()
-  const parser = new DOMParser()
-  const doc = parser.parseFromString(text, 'text/html')
-  sidebarTemplateContent = doc.getElementById('sidebar-template').content
+const response = await fetch('./sidebar.html')
+const text = await response.text()
+const template = document.createElement('template')
+template.innerHTML = text
 
-  customElements.define('sidebar-nav', SidebarNav)
-}
+const style = document.createElement('style')
+style.textContent = '@import url("./sidebar.css");'
+template.content.appendChild(style)
```

Comment:
```suggestion
document.head.appendChild(style)
```

We should append to the head instead